src/app/clusters/door-lock-server/door-lock-server.cpp

```diff
@@ -1242,6 +1242,7 @@ bool DoorLockServer::OnFabricRemoved(chip::EndpointId endpointId, chip::FabricIn
     {
         ChipLogError(Zcl, "[OnFabricRemoved] Unable to cleanup fabric from users - internal error [endpointId=%d,fabricIndex=%d]",
                      endpointId, fabricIndex);
+        return false;
```

Comment:
Nit comment. I am unfamiliar with this overall, so feel free to ignore and resolve this comment if you don't think it applies or makes sense

In here, and the other methods `clearFabricFromUsers`, and `clearFabricFromCredentials`, does it make sense to start of with `bool returnValue = true` at the top and set it to false in these if statements and just `return returnValue` so we at least attempt to clear other things even if we fail on one thing?

Reply:
Thanks, good idea! I'm just unsure if it actually makes sense to keep going if calls to the app fail. We just need to choose the strategy and apply it everywhere. I am currently not ready to make this decision, maybe it makes sense to create an issue for this.

Reply:
Sounds good to me. You can resolve this comment and merge the PR, and create the issue